database/init.sql

```diff
@@ -381,4 +382,4 @@ on
   y_main_aa_sequence,
   y_main_aa_sequence_columnar,
   access_key
-to y_user;
+to lapis_proc;
```

Comment:
This bloc can be removed because lapis_proc is already getting all the permission it needs [here](https://github.com/GenSpectrum/LAPIS/blob/7bae322f06ffb66788631d6c2b5c78e8fc21dae3/database/users.sql#L6)